tests/helpers_tests/test_condition.py

```diff
@@ -12,122 +12,587 @@
 # limitations under the License.
 
 import mock
+from six import PY2, PY3
 
 from optimizely.helpers import condition as condition_helper
 
 from tests import base
 
+if PY3:
+  def long(a):
+    pass
```

Comment:
```suggestion
    raise NotImplementedError('Tests should only call `long` if running in PY2')
```

Reply:
(I'm guessing you only need to define `long` because the interpreter can't determine that the calls to `long` are unreachable in PY3?)

Reply:
Yeah. Exactly